preload.php

```diff
@@ -314,7 +314,7 @@
 require_once __DIR__ . '/vendor/phpstan/phpdoc-parser/src/Ast/PhpDoc/VarTagValueNode.php';
 require_once __DIR__ . '/vendor/phpstan/phpdoc-parser/src/Ast/Type/ArrayShapeItemNode.php';
 require_once __DIR__ . '/vendor/phpstan/phpdoc-parser/src/Ast/Type/ArrayShapeNode.php';
-require_once __DIR__ . '/vendor/phpstan/phpdoc-parser/src/Ast/Type/ArrayShapeUnsealedTypeNode.php';
+//require_once __DIR__ . '/vendor/phpstan/phpdoc-parser/src/Ast/Type/ArrayShapeUnsealedTypeNode.php';
```

Comment:
preload.php is generated by build-preload.php, this commented line will be replaced on per week generated

https://github.com/rectorphp/rector-src/blob/main/build/build-preload.php

https://github.com/rectorphp/rector-src/blob/96b585bd2640cb09ef79271907f444f3218c42b2/.github/workflows/weekly_pull_requests.yaml#L28